Puyolib/Game.cpp

```diff
@@ -56,6 +59,8 @@ Game::~Game()
 	delete m_statusFont;
 	delete m_data->front;
 	delete m_data;
+	delete m_debug;
+	m_debug = nullptr;
```

Comment:
I can see why you would do this, but it is probably a bit unnecessary, since we don't clear any of the other pointers after deleting them in the dtor. There's not a whole lot involving this object that can happen after this that would be valid/defined behavior anyway.

Reply:
Corrected, removed. It was originally there to check for double-free and I forgot to remove it.